### PR TITLE
Add Helm Chart

### PR DESCRIPTION
**Pull request by pperzyna**
## Description

Mandatory, provide a detailed description of the changes.

## Why

Optional i.e. "Why I have chosen package X, and not alternative package Y, Z"

## AC

Copy and paste from task

## Source

Link to story/task/bug

## Type

- [ ] Bug Fix
- [ ] Feature Addition
- [ ] Refactor
- [ ] HotFix

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: charts/hat/values.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+      - path: /
+        backend:
+          serviceName: chart-example.local
```
Comment (xavji) [inline charts/hat/values.yaml]: should this not be changed ?

File: charts/hat/values.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+env:
+  config: []
+  secret:
+    enabled: false
+    name: ""
+
+service:
+  type: ClusterIP
+  port: 9000
```
Comment (whiteshadow-gr) [inline charts/hat/values.yaml:47]: I might be wrong but we use 8080 currently? 🤔 No idea if it should be 8080 or 9000 just a heads up

**Review by whiteshadow-gr — APPROVED** [review]

File: charts/hat/values.yaml (hunk 2 above)
Comment (pperzyna) [reply charts/hat/values.yaml:47]: we use 9000

File: charts/hat/values.yaml (hunk 1 above)
Comment (pperzyna) [reply charts/hat/values.yaml]: those are just sample values, that will be override per env

**Review by whiteshadow-gr — APPROVED** [review]

**Pull request merged**